### ISSUE TITLE
Fehler beim Aufrufen von "Infos zum Lager"

### ISSUE THREAD

**Issue by tierriminator** [PfadiAngenstein/ecamp2#19]
Beim Klick auf "Infos zum Lager" erscheint der Fehler `{"error":false,"msg":"[]"}`

**Comment by andystoeggle** [comment]
wie erscheint der fehler? als `alert` oder in der konsole? Prinzipiell beudeutet diese nachricht, dass kein fehler aufgetreten ist (`error=false`) und keine dateien gefunden wurden (`msg=[]`) somit steht die meldung selbst nicht für einen fehler.

**Comment by tierriminator** [comment]
Inzwischen funktioniert es zumindest bei mir. Die obige Nachricht wurde anstelle der zu erwartenden Seite ausgegeben (sprich genau so sah ein Nutzer die Meldung anstelle der Seite).

**Comment by andystoeggle** [comment]
unschön. evtl hast du direkt auf die funktion (`action_show_files.php`) zugegriffen? kann der issue geschlossen werden?

**Comment by tierriminator** [comment]
Nein, es geschah wirklich einfach beim Klick auf den Link „Infos zum Lager“ solange die Commits seit dem Eröffnen dieses Issues nicht kontrolliert wurden, ob der Bug behoben wurde, sollte das Issue nicht geschlossen werden.

**Comment by andystoeggle** [comment]
tritt der fehler in diesem falle immernoch auf?
